Lib/_pyrepl/console.py

```diff
@@ -174,7 +174,7 @@ def _excepthook(self, typ, value, tb):
 
     def runsource(self, source, filename="<input>", symbol="single"):
         try:
-            tree = ast.parse(source)
+            tree = self.compile.compiler(source, filename, symbol, ast.PyCF_ONLY_AST)
```

Comment:
```suggestion
            tree = self.compile.compiler(source, filename, "exec", ast.PyCF_ONLY_AST, incomplete_input=False)
```

Reply:
@Wulian233, could you apply this suggestion? The other problems should be handled separately.

Reply:
> @Wulian233, could you apply this suggestion? The other problems should be handled separately.

Sure!